content/en/docs/setup/production-environment/tools/kops.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: Installing Kubernetes on AWS with kops
+title: Installing Kubernetes with kops
```

Comment:
If renaming, how about renaming to `kops-aws.md`?

Reply:
Not sure if we want any vendor specific docs in future at all. I will leave this as is for now.